model/src/main/kotlin/OrtResult.kt

```diff
@@ -237,6 +237,25 @@ data class OrtResult(
         return vendorPackages
     }
 
+    /**
+     * Returns the path of the folder that contains the definition file of the [project], relative to the root VCS.
+     */
+    fun getCompleteDefinitionFilePath(project: Project): String {
+        val vcsPath = repository.getVcsPath(project.vcsProcessed)
+
+        requireNotNull(vcsPath) {
+            "The VCS ${project.vcs} is not contained in the repositories of this ORT result: $repository"
```

Comment:
As `${project.vcs}` already expands to "VcsInfo(...)" maybe just:

    "${project.vcs} is not contained in $repository."

Reply:
Partly done, but I have also added the id of the project to the message to provide more context.